lib/rubocop.rb

```diff
@@ -627,3 +628,7 @@
 require_relative 'rubocop/remote_config'
 require_relative 'rubocop/target_ruby'
 require_relative 'rubocop/yaml_duplication_checker'
+
+unless File.exist?("#{__dir__}/../Gemfile") # Check if we are a gem
```

Comment:
Would `File.exist?(Bundler.default_gemfile)` be better here, since the gemfile does not have to be actually named `Gemfile` these days?

Reply:
I guess since you're just looking for rubocop's Gemfile itself it's ok to hardcode it.

Reply:
I didn't know that. I'm not sure if `Bundler` is always required, so I'll leave it as is. The potential impact would be for someone who is developping on `RuboCop`, and somehow went out of their way to not use our `Gemfile` and deleting it to boot...
Thanks for bringing it up though.

Reply:
Btw, isn't it better to check for the gemspec instead?

Reply:
No problem; #8645 should merge soon.